packages/base-styles/_variables.scss

```diff
@@ -56,6 +56,7 @@ $mobile-header-toolbar-height: 44px;
 $mobile-floating-toolbar-height: 44px;
 $mobile-floating-toolbar-margin: 8px;
 $mobile-color-swatch: 48px;
+$header-toolbar-min-width: 335px;
```

Comment:
@jeyip I know this value is iPhone friendly, but it doesn't appear to work as intended as the mobile experience is otherwise broken. The variable is also used only in this PR and no-where else. I'd recommend not adding this variable, and instead either use `$break-mobile` from _breakpoints.scss so as to avoid fragmenting the responsive dev.

We also have some good mixins that help mobile dev, such as `@include break-small() { ... }` to abstact and keep things in sync.

Reply:
The 335px was meant to ensure a certain width for the toolbar on either side of the header used [here](https://github.com/WordPress/gutenberg/pull/25884/files/4a2937aca4377f3eda568b93d865980e7e72f893#diff-0cda043c56d581dee157884a2abbedb914c3e24d9712cf0317fea8de5a4bf1a9R16-R20) and [here](https://github.com/WordPress/gutenberg/pull/25884/files/4a2937aca4377f3eda568b93d865980e7e72f893#diff-0cda043c56d581dee157884a2abbedb914c3e24d9712cf0317fea8de5a4bf1a9R34-R40). The min width prevented the toolbars from collapsing as the browser viewport was shrunk horizontally. Giving this more thought, it should absolutely be a desktop specific affordance. 

<img width="1344" alt="Screen Shot 2020-10-26 at 3 21 32 PM" src="https://user-images.githubusercontent.com/5414230/97234987-2bfa6080-179f-11eb-842c-178e4b729c5d.png">

I'll open another PR incorporating the break point mixins.

Reply:
Ah! Thank you for the explanation. I initially pondered where those 335px came from, because it seems a relatively arbitrary number given the base8 grid most of the UI uses. I then thought — maybe it's that somewhat odd iphone resolution which I _remembered_ as 335px (iPhone 6: 750 x 1334, which makes it 375 points horizontally) but as you can see, I was wrong.

With your explanation, though, I think a different approach is needed for min-width. Longer term I think we need to stack the toolbar if there isn't space, but nearer term I think smaller steps can be taken.

> I'm happy to open another PR incorporating the break point mixins.

#26021 does that, and but also hides some buttons, and minimizes labels, as the viewport approaches mobile breakpoints. It's only a first step, and more are needed, but it's a good step. It's currently blocked on the white screen as shown in the intro of the PR, which happens because the left sidebar is always output, regardless of open or not. I could use help with that if you have bandwidth!

> I'm not quite sure I'm tracking here. I think the name of the variable may be misleading, because it sounds like we see the 335px as an iphone friendly breakpoint?

To be specific, we should only be creating new variables in the _variables.scss file if they are meant to be used widely and in multiple places. If it's just for the site editor as is currently the case, I'd just type the content of the variable directly where needed, or just register the variable directly in those files.

I'd look at finding a different approach than the min-width, though (not urgently) it seems a half-step towards solving a problem that needs a more foundational mobile improvement. The work could probably coincide with enforcing of the stacking top toolbar, so we fix this:

<img width="1406" alt="Screenshot 2020-10-27 at 09 55 55" src="https://user-images.githubusercontent.com/1204802/97278966-a926e380-183a-11eb-9cee-a9441f86c12c.png">

I know it's beta! I know there's a lot of work to be done. I'm just:

- excited about the site editor
- of the belief that if we test mobile in every PR, even if it's not superb, we minimize future headaches

Reply:
I added a [commit](https://github.com/WordPress/gutenberg/pull/26021/commits/b4746b5bc04e676134df1a714d51896fff399668) and [a comment](https://github.com/WordPress/gutenberg/pull/26021#issuecomment-717095993) to #26021 that potentially address the issues you bring up here.